src/build.py

```
@@ -0,0 +1,31 @@
+import logging
+from subprocess import CompletedProcess
+
+from util import run_bash, find_config
+
+def build_repo(repo_data, configs, result, args):
+    # Find the build config for this repo
+    build_config = find_config(configs, repo_data["app"], repo_data["target_model"])
+    if not build_config:
+        return
```

raise error here instead of silently returning.